apps/studio/components/interfaces/DataWarehouse/CreateWarehouseAccessToken.tsx

```diff
@@ -0,0 +1,69 @@
+import { set } from 'lodash'
+import React from 'react'
+import { Button, Input, Modal } from 'ui'
+import { FormItemLayout } from 'ui-patterns/form/FormItemLayout/FormItemLayout'
+
+type Props = {
+  onSubmit: (values: { description: string }) => Promise<void>
+}
+
+const CreateWarehouseAccessToken = (props: Props) => {
```

Comment:
for consistency: lets de-structure the props

`const CreateWarehouseAccessToken = ({ onSubmit }: Props) => {...}`

Reply:
we also usually name the Props like `ComponentNameProps`